addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -41,12 +49,25 @@ export class ProjectTaskFormController extends FormController {
         const versionedFieldName = 'description';
         const historyMetadata = record.data["html_field_history_metadata"]?.[versionedFieldName];
         if (!historyMetadata) {
+            this.notifications.add(
+                escape(_t(
+                    "The task description lacks any past content that could be restored at the moment."
+                ))
+            );
             return;
         }
 
         this.dialogService.add(
             HistoryDialog,
             {
+                title: _t("Task Description History"),
+                noContentHelper: markup(
+                    `<span class='text-muted fst-italic'>${escape(
+                        _t(
+                            "The task description was empty at the time."
+                        )
+                    )}</span>`
+                ),
```

Comment:
Hello @odoo/rd-security 🙂 

Can we have an override?

We use markup function and we escape the content translated given to the markup. 🙂

Thanks in advance! 🙂 

Have a nice day! 😁